developing.adoc

```diff
@@ -0,0 +1,45 @@
+
+
+= Developing {project}
```

Comment:
`{project}` is not defined anywhere.

As I understand it, this project will only have a single name in any case. Considering that, I believe we can simply use a `{crc}` attribute (simply to make referring to the name simpler and less likely to be misrepresented).

Reply:
@robin-owen Once we have the a release we need to release the docs in a similar way as Minishift. So we need replace this when generating the HTML files. @code-ready/crc-devel  WDYT
I am not sure how to handle the situation now as we will not be releasing this documentation as HTML  till the release. So wondering if we can handle the attribute for GitHub rendering.

Reply:
@robin-owen I think we should create an issue for this topic and discuss it there. This discussion should also happen as part of overall documentation discussion.